tests/manual-test-cases/Group5-Functional-Tests/5-12-Multiple-VLAN.robot

```diff
@@ -1,12 +1,14 @@
 *** Settings ***
 Documentation  Test 5-12 - Multiple VLAN
 Resource  ../../resources/Util.robot
-Suite Setup  Create a Simple VC Cluster  multi-vlan-1  cls
-Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup
+Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
 Test Teardown  Cleanup VIC Appliance On Test Server
 
 *** Test Cases ***
 Test1
+    ${esx1}  ${esx2}  ${esx3}  ${vc}  ${vc-ip}=  Create a Simple VC Cluster  multi-vlan-1  cls
```

Comment:
Sure, I like the idea on the thin wrapper, but can I make it return values and use it from the Suite Setup?